README.md

```diff
@@ -20,6 +20,10 @@ To include `defmt` in your existing project, follow our [Application Setup guide
 
 [Application Setup guide]: https://defmt.ferrous-systems.com/setup-app.html
 
+## MSRV
+`defmt` always compiles on the [latest `stable` rust release](https://github.com/rust-lang/rust/releases/latest). This is enforced by our CI building and testing against this version. <br>
```

Comment:
nit: can we avoid the HTML element here?

Reply:
Yees, but then we either have two spaces, which is bad because it's hard to see, or a newline, but this actually appears differently in the rendered version. But in the end it probably doesn't matter.